util/file_checksum_helper.h

```diff
@@ -16,24 +16,24 @@ namespace ROCKSDB_NAMESPACE {
 
 // This is the class to generate the file checksum based on Crc32. It
 // will be used as the default checksum method for SST file checksum
-class FileChecksumFuncCrc32c : public FileChecksumFunc {
+class FileChecksumGenCrc32c : public FileChecksumGenerator {
  public:
-  std::string Extend(const std::string& init_checksum, const char* data,
-                     size_t n) override {
-    assert(data != nullptr);
-    uint32_t checksum_value = StringToUint32(init_checksum);
-    return Uint32ToString(crc32c::Extend(checksum_value, data, n));
+  FileChecksumGenCrc32c(const FileChecksumGenOptions& options) {
+    is_inintilized_ = false;
+    file_name_ = options.file_name;
   }
 
-  std::string Value(const char* data, size_t n) override {
+  void Extend(const char* data, size_t n) override {
     assert(data != nullptr);
-    return Uint32ToString(crc32c::Value(data, n));
+    if (is_inintilized_ == false) {
+      checksum_ = crc32c::Value(data, n);
```

Comment:
Here is the implementation of crc32c::Value():
```
inline uint32_t Value(const char* data, size_t n) {
  return Extend(0, data, n);
}
```
I don't think we need this `if` and `is_inintilized_`.

Reply:
@siying  Thanks for pointing out. I will change accordingly.